packages/react/src/hooks/useMerged.js

```diff
@@ -0,0 +1,8 @@
+import { useMemo } from 'react';
+import { merge } from 'lodash-es';
```

Comment:
I am not necessarily opposed to keeping this but is there something `merge` does that we can't achieve with just spreading the args onto an object literal?

Reply:
The main thing is that `merge` will only replace the existing item if it's undefined whereas spreading will replace anything that's falsy. Using merge allows a parent to set a string as `""` (when we don't want to show a label, for example) without a child replacing it with it's default.